pkg/multimodelconfig/configmap.go

```diff
@@ -0,0 +1,78 @@
+package multimodelconfig
+
+import (
+	"encoding/json"
+	"k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+type MultiModelConfig struct {
+	fileName    string
+	fileContent ModelDefinition
+}
+
+type ModelDefinition struct {
+	StorageUri string `json:"storageUri"`
+	Framework  string `json:"framework"`
+	Memory     string `json:"memory"`
```

Comment:
Isn't memory another struct? i.e. cpu, gpu, memory?

Reply:
No, we decided to change it to memory only in my previous pr

Reply:
the type should be `resource.Quantity` not string

Reply:
also this struct seems to be the same as `ModelSpec` defined on `TrainedModel`, should we use that instead?

Reply:
> No, we decided to change it to memory only in my previous pr

oops sorry I changed it to ```Memory resource.Quantity `json:"memory,omitempty"`` `` later

Reply:
Discussed with @yuzisun offline, we can keep the ModelDefinition struct but we don't need to put the `Memory` field since model servers does not need this information to load/unload. The memory field will be used by the TrainedModel controller to decide if the parent inferenceservice has enough memory.

Reply:
I agree with removing `memory` for now